docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
 By default, the `journal` is located in a subdirectory of the `archive`.
 To achieve the best cost vs speed tradeoff, you can separate the journal from the archive.
 To achieve that set the database helm template values `database.sm.noHotCopy.journalPath.enabled` and `database.sm.hotCopy.journalPath.enabled` to `true` and configure it with the desired persistence settings.
 
 Kubernetes stateful sets volume mounts are immutable and as such, the setting can not be changed easily on an existing database.
-More on upgrades below [Upgrading existing domains](#upgrading-existing-domains).
 
 ## Upgrading existing domains
-TODO
\ No newline at end of file
+
+### Overview
```

Comment:
This process is relying on the syncing to recover with the journals in the correct locations.  We might want to comment on considerations of the time it may take to sync very large archives (and / or offer an alternative in those situations)?

Reply:
Something like?

This migration process will resync new Storage Managers from the remaining Storage Managers.
The time it takes to resync a Storage Manager from scratch depends on the size of the archive and on the load of the system.
When in doubt whether you can perform this operation safely, please contact [NuoDB Support](mailto:support@nuodb.com).

Reply:
I have also filed a product improvement to make sure that we can mutate the journal location.